app/src/components/TimestampOrderBookScroller.tsx

```diff
@@ -38,19 +40,25 @@ class TimestampOrderBookScroller extends Component<Props> {
     }
 
     shouldComponentUpdate(nextProps, nextState, nextContext) {
-        const { lastSodOffset } = this.props;
+        const { lastSodOffset, loading } = this.props;
+        if (loading !== nextProps.loading) {
+            return true;
+        }
         if (lastSodOffset && nextProps.lastSodOffset) {
             return (lastSodOffset.neq(nextProps.lastSodOffset));
         }
         return true;
     }
 
     componentDidUpdate(prevProps, prevState, snapshot) {
-        const { listItems } = this.props;
+        const { listItems, loading } = this.props;
 
         if (!listItemsEquals(prevProps.listItems || {}, listItems || {})) {
             this.handleScrollToTopOfTheBook();
         }
+        if (loading !== prevProps.loading) {
+            this.forceUpdate();
```

Comment:
Is this `forceUpdate()` necessary ? can you try without it, since a change in prop should re-render the component.

Reply:
Did not remove it when finally pushing. Originally was testing something out and was not working as intended.